src/common/models/measure/measure.ts

```diff
@@ -224,22 +232,34 @@ export class Measure implements Instance<MeasureValue, MeasureJS> {
     return this.formatFn(datum[this.name] as number);
   }
 
-  public change(propertyName: string, newValue: any): Measure {
+  public change(propertyName: string, newValue: any): this {
```

Comment:
Ok so in: https://www.typescriptlang.org/docs/handbook/advanced-types.html read the section "Polymorphic this types".